Generators/src/GeneratorFromFile.cxx

```diff
@@ -245,8 +246,8 @@ bool GeneratorFromO2Kine::importParticles()
 
       LOG(debug) << "Putting primary " << pdg;
 
-      mParticles.push_back(TParticle(pdg, wanttracking, m1, m2, d1, d2, px, py, pz, e, vx, vy, vz, vt));
-      mParticles.back().SetUniqueID((unsigned int)t.getProcess()); // we should propagate the process ID
```

Comment:
what about the physics process?

Reply:
that was kicked out by accident, back in now